### DISCUSSION TITLE
Populating the Vulnerabilities list in KBOM?

### DISCUSSION THREAD

**Discussion by tspearconquest** [aquasecurity/trivy#5582] [Q&A] [answered]
### Question

I just generated 2 KBOM files; one from a local Kind cluster on my mac, and one from an AKS cluster.

While viewing the files, I noticed that the `Vulnerabilities` array at the bottom is empty. Is there any way to populate the array with Trivy at this time?

I used Trivy version 0.47.0 to generate the KBOMs.

### Target

Kubernetes

### Scanner

None

### Output Format

CycloneDX

### Mode

Standalone

### Operating System

macOS Sonoma 14.1.1

### Version

```bash
Version: 0.47.0
Vulnerability DB:
  Version: 2
  UpdatedAt: 2023-11-14 12:12:55.648130557 +0000 UTC
  NextUpdate: 2023-11-14 18:12:55.648130307 +0000 UTC
  DownloadedAt: 2023-11-14 16:48:29.429133 +0000 UTC
Policy Bundle:
  Digest: sha256:714272c66dfff71107a717b64960f00d54883d7268e30e321cbe89c9ecac0f93
  DownloadedAt: 2023-03-28 15:00:59.31549 +0000 UTC
```

**Comment by chen-keinan** [comment]
@tspearconquest you'll need to run it with two steps :

- `trivy k8s --format cyclonedx cluster -o kbom.json`
-  `trivy sbom kbom.json`

**Reply by tspearconquest** [reply]
Yes, this is what I have done, and the vulnerabilities list is empty in `kbom.json`, however the second command outputs the vulnerabilities in table format when I scan the `kbom.json`

How can I get the listed vulnerability from the `trivy sbom` command to be output into the `kbom.json` file's `vulnerabilities` object?

**Reply by itaysk** [reply, marked as answer]
```
trivy sbom kbom.json --format cyclonedx --output kbom-with-vulns.cdx.json
```

**Reply by tspearconquest** [reply]
It works! Fantastic, thank you!
